zebrad/tests/acceptance.rs

```diff
@@ -1780,7 +1784,8 @@ fn lwd_integration() -> Result<()> {
 
 /// Make sure `zebrad` can sync from peers, but don't actually launch `lightwalletd`.
 ///
-/// This test only runs when the `ZEBRA_CACHE_DIR` env var is set.
+/// This test only runs when a persistent cached state is configured
```

Comment:
This and similar updates below look wrong, because it'll use the default state cache dir path if none is configured.

Nitpick:

```suggestion
/// This test only runs when a persistent cached state directory path is configured
```